sample/src/test/kotlin/com/slack/circuit/sample/petlist/PetListPresenterTest.kt

```diff
@@ -41,7 +41,7 @@ class PetListPresenterTest {
   @Test
   fun `present - emit loading state then no animals state`() = runTest {
     val repository = TestRepository(emptyList())
-    val presenter = PetListPresenter(navigator, repository)
+    val presenter = PetListPresenter(navigator, PetListScreen(Gender.ALL, Size.ALL), repository)
```

Comment:
Since these are just using the default values, can we just use this?

```suggestion
    val presenter = PetListPresenter(navigator, PetListScreen(), repository)
```

Reply:
True, addressed here https://github.com/slackhq/circuit/pull/101/commits/89de7f6d29b498d957c17c4ec559be364ecd09c1